commands/serve/web/components/property-panel/DataCube.jsx

```diff
@@ -25,12 +26,13 @@ const getValue = (data, reference, defaultValue) => {
   return dataContainer;
 };
 
-export default function HyperCube({ setProperties, target, properties }) {
+export default function DataCube({ setProperties, target, properties }) {
+  const createHandler = target.propertyPath.match('/qHyperCube') ? hcHandler : loHandler;
```

Comment:
Haven't pieced through the whole process, but shouldn't there be some more checks for things like this? So that we throw some useful error messages around.

Reply:
We do this in qae.js (lines 69-72):
```
  if (/\/(qHyperCube|qListObject)$/.test(layoutPath) === false) {
    const d = layoutPath.includes('/qHyperCube') ? 'qHyperCubeDef' : 'qListObjectDef';
    throw new Error(`Incorrect definition for ${d} at ${propertyPath}`);
  }
```